src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,19 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/types';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+
+type TodoInfoProps = {
+  todo: Todo;
+};
+export const TodoInfo: React.FC<TodoInfoProps> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```
and just add the check here